### PR TITLE
RWA-2927: Reconfiguration batch job schedule change

### PR DESCRIPTION
**Pull request by kvvasudev**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RWA-2927

### Change description ###

Change cron to run every 1 minute for Reconfiguration job

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: apps/wa/wa-task-batch-reconfiguration-failure/ithc/00.yaml [hunk 1]
```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   values:
     job:
-      schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
+      schedule: "1-60/2 * * * *"
```
Comment (raghera) [inline apps/wa/wa-task-batch-reconfiguration-failure/ithc/00.yaml:8]: Aren't mins supposed to be from 0 to 59?

Comment (raghera) [reply apps/wa/wa-task-batch-reconfiguration-failure/ithc/00.yaml:8]: Can review all the rest too I guess.

**Review by raghera — COMMENTED** [review]
I thought the change was to change it in prod not ithc?  I think we just need to do it in prod.

**Review by raghera — APPROVED** [review]

File: apps/wa/wa-task-batch-reconfiguration-failure/ithc/00.yaml (hunk 1 above)
Comment (kvvasudev) [reply apps/wa/wa-task-batch-reconfiguration-failure/ithc/00.yaml]: Addressed the change

**Pull request merged**